rustfmt.toml

```diff
@@ -0,0 +1,6 @@
+edition = "2018"
+max_width = 100
```

Comment:
This seems to be the default. Should we only overwrite values that we want to not be the default?